### ISSUE TITLE
[Request] Builtin volume, brightness notifications

### ISSUE THREAD

**Issue by jackghg** [dunst-project/dunst#1052]
I know that there are fancy scripts and tweaks to enable volume and brightness notifications, but I'd like these builtin.

**Comment by jackghg** [comment]
@fwsmit right, it wouldnt be easy. A monitor process wouldnt be lightweight.
A way could be to add the volume keys in the conf. But it would be a similar job to add the keys to xbindkeys...
Maybe optimize dunstify ? make a preset for it ?

**Comment by fwsmit** [comment]
I think these scripts shouldn't be run by dunst at all. The best we can do is include them in the contrib directory and add documentation on the wiki. Feel free to contribute to that.

**Comment by jackghg** [comment]
Basicly I sayd that its better to use xbindkeys or the DE hotkey utility.
But a preset for dunstify could be done to simplify and show a better looking vol/brightness notification

**Comment by fwsmit** [comment]
Ah alright, we could add some configuration options to the example dunstrc for brightness and volume notifications.

**Comment by fwsmit** [comment]
I'm closing this issue, as builtin volum/brightness notifications probably won't be a thing
